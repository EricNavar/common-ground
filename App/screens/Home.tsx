/* eslint-disable @typescript-eslint/no-unused-vars */
import axios from 'axios';
import React from 'react';
import { Text, View, StyleSheet, ScrollView, Button } from 'react-native';
import { useIsFocused } from '@react-navigation/native';
import { GroupBubbles } from '../components/GroupBubbles';
import { RestaurantBubble } from '../components/RestaurantBubble';
import { SERVER_URI } from '../Config';
import { parties, saulProfile } from '../data/dummyUsers';
import { Storage } from '../data/Storage';

const styles = StyleSheet.create({
  welcome: {
    fontSize: 28,
    fontWeight: 'bold',
    marginBottom: 20,
    paddingHorizontal: 20,
    paddingTop: 20,
  },
  startNewTableWrapper: {
    color: 'black',
    marginLeft: 'auto',
    marginRight: 'auto',
    borderRadius: 6,
    marginTop: 8,
    marginBottom: 20,
  },
  sectionTitle: {
    fontSize: 20,
    fontWeight: 'bold',
    marginBottom: 12,
    paddingHorizontal: 20,
    marginRight: 'auto',
    marginVertical: 8,
  },
  row: {
    display: 'flex',
    flexDirection: 'row',
    overflowX: 'scroll',
    flexWrap: 'nowrap',
    marginTop: 8,
  },
  sectionHeaderContainer: {
    display: 'flex',
    flexDirection: 'row',
    overflowX: 'scroll',
    flexWrap: 'nowrap',
    marginTop: 20,
  },
  restaurantBubble: {
    marginLeft: 20,
  },
  verticalSpace: {
    height: 20,
  },
});

type HomeProps = {
  navigation: {
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    navigate: any;
  };
};

function Home(props: HomeProps) {
  
  const getProfileInfo = async () => {
    //Get profile info from async storage
    const profile = await Storage.get('profile');
    if (profile) {
      const profileInfo = JSON.parse(profile);
      setFirstName(profileInfo.firstName);
      setLastName(profileInfo.lastName);
      setEmail(profileInfo.email);
      setLocation(profileInfo.location);
    } else {
      //TODO: redirect to login page
    }
  };

  const getParties = async () => {
    axios.get(`${SERVER_URI}/groups`, {
      params: {
        name: true,
      },
    }).then((res) => {
      console.log(res.data);
      // setParties(res.data);
    }).catch((err) => {
      console.log(err);
    });
  };

  const [firstName, setFirstName] = React.useState<string>(saulProfile.firstName);
  const [lastName, setLastName] = React.useState<string>(saulProfile.lastName);
  const [email, setEmail] = React.useState<string>(saulProfile.email);
  const [location, setLocation] = React.useState<string>(saulProfile.location);

  const isFocused = useIsFocused();

  React.useEffect(() => {
    getProfileInfo();
    getParties(); 
  }, [isFocused]);

  const restaurant = {
    name: 'Los Pollos Hermanos',
    thumbnail:
      'https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200',
  };

  const onClickRestaurant = () => {
    props.navigation.navigate('Restaurant');
  };

  const onClickGroup = () => {
    props.navigation.navigate('Group Details');
  };

  const recentlyVisited = [restaurant, restaurant, restaurant];

  const onPressStartANewTable = () => {
    props.navigation.navigate('Start New Table');
  };

  const onPressSeeAllSaved = () => {
    props.navigation.navigate('Restaurant List');
  };

  return (
    <ScrollView>
        <Text style={styles.welcome}>
          Welcome back, {firstName}!
        </Text>
        <View style={styles.startNewTableWrapper}>
          <Button title='Start a new table' color='#FF6D6E' onPress={onPressStartANewTable}/>
        </View>
        <Text style={styles.sectionTitle}>Recently Visited</Text>
        <ScrollView style={styles.row} horizontal={true}>
          {recentlyVisited.map((restaurant, index) => 
            <RestaurantBubble
              key={index}
              {...restaurant}
              onPress={onClickRestaurant}
              style={styles.restaurantBubble}
            />
          )}
          <RestaurantBubble {...restaurant} onPress={onClickRestaurant} style={styles.restaurantBubble}/>
          <RestaurantBubble {...restaurant} onPress={onClickRestaurant} style={styles.restaurantBubble}/>
        </ScrollView>
        <View style={styles.sectionHeaderContainer}>
          <Text style={styles.sectionTitle}>Recent Parties</Text>
        </View>
        <ScrollView style={styles.row} horizontal={true} contentContainerStyle={{alignItems:'flex-start'}}>
          {parties.map((party, index) => 
            <GroupBubbles
              members={party.members}
              name={party.name}
              onClick={onClickGroup}
              style={styles.restaurantBubble}
              key={index}
            />
          )}
        </ScrollView>
        <View style={styles.sectionHeaderContainer}>
          <Text style={styles.sectionTitle}>Saved restaurants</Text>
          <Button
            onPress={onPressSeeAllSaved}
            title="See all"
            color="#ff6e6e"
          />
        </View>
        <ScrollView style={styles.row} horizontal={true}>
          {recentlyVisited.map((restaurant, index) => 
            <RestaurantBubble
              key={index}
              {...restaurant}
              onPress={onClickRestaurant}
              style={styles.restaurantBubble}
            />
          )}
        </ScrollView>
        <View style={styles.verticalSpace}/>

    </ScrollView>
  );
}

export { Home };
