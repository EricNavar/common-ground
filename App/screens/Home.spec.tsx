import { Home } from './Home';
import { render } from '@testing-library/react-native';
import React from 'react';

describe('Homee', () => {
    const dummyNavigation = {
        navigate: '',
    };
    it('Renders Home correctly', () => {
        expect(true).toBe(true);
        return;
        const result = render(<Home navigation={dummyNavigation} />);

        expect(result.toJSON()).toMatchSnapshot();
    });
});
