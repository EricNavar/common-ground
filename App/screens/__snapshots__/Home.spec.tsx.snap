// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homee Renders Home correctly 1`] = `
<RCTScrollView>
  <View>
    <Text
      style={
        {
          "fontSize": 28,
          "fontWeight": "bold",
          "marginBottom": 20,
          "paddingHorizontal": 20,
          "paddingTop": 20,
        }
      }
    >
      Welcome back, 
      Saul
      !
    </Text>
    <View
      style={
        {
          "borderRadius": 6,
          "color": "black",
          "marginBottom": 20,
          "marginLeft": "auto",
          "marginRight": "auto",
          "marginTop": 8,
        }
      }
    >
      <View
        accessibilityRole="button"
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            [
              {},
            ]
          }
        >
          <Text
            style={
              [
                {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "margin": 8,
                  "textAlign": "center",
                },
                {
                  "color": "#FF6D6E",
                },
              ]
            }
          >
            Start a new table
          </Text>
        </View>
      </View>
    </View>
    <Text
      style={
        {
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 12,
          "marginRight": "auto",
          "marginVertical": 8,
          "paddingHorizontal": 20,
        }
      }
    >
      Recently Visited
    </Text>
    <RCTScrollView
      horizontal={true}
      style={
        {
          "display": "flex",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "marginTop": 8,
          "overflowX": "scroll",
        }
      }
    >
      <View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
      </View>
    </RCTScrollView>
    <View
      style={
        {
          "display": "flex",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "marginTop": 20,
          "overflowX": "scroll",
        }
      }
    >
      <Text
        style={
          {
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 12,
            "marginRight": "auto",
            "marginVertical": 8,
            "paddingHorizontal": 20,
          }
        }
      >
        Recent Parties
      </Text>
    </View>
    <RCTScrollView
      contentContainerStyle={
        {
          "alignItems": "flex-start",
        }
      }
      horizontal={true}
      style={
        {
          "display": "flex",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "marginTop": 8,
          "overflowX": "scroll",
        }
      }
    >
      <View>
        <View
          style={
            [
              {
                "justifyContent": "center",
              },
              {
                "marginLeft": 20,
              },
            ]
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#ddd",
                "borderRadius": 50,
                "height": 100,
                "marginBottom": 10,
                "opacity": 1,
                "padding": 20,
                "position": "relative",
                "width": 100,
              }
            }
          >
            <Image
              source={
                {
                  "uri": "https://compote.slate.com/images/fb69a16d-7f35-4103-98c1-62d466196b9a.jpg?width=1200",
                }
              }
              style={
                [
                  {
                    "borderRadius": 22.5,
                    "bottom": 5,
                    "height": 45,
                    "right": 5,
                    "width": 45,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
            <Image
              source={
                {
                  "uri": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQS7_zsiM5wfE2uac1B7Xk-jdVGpCKzrQqEwOaK9ak60O1iI4Don3BcWR_sOsumMQetlSw&usqp=CAU",
                }
              }
              style={
                [
                  {
                    "borderRadius": 27.5,
                    "height": 55,
                    "right": 5,
                    "top": 5,
                    "width": 55,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
            <Image
              source={
                {
                  "uri": "https://static.wikia.nocookie.net/breakingbad/images/5/54/BCS_S3_GusFring.jpg/revision/latest?cb=20170327185354",
                }
              }
              style={
                [
                  {
                    "borderRadius": 30,
                    "height": 65,
                    "top": 10,
                    "width": 65,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
          </View>
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "width": 100,
              }
            }
          >
            The boys
          </Text>
        </View>
        <View
          style={
            [
              {
                "justifyContent": "center",
              },
              {
                "marginLeft": 20,
              },
            ]
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#ddd",
                "borderRadius": 50,
                "height": 100,
                "marginBottom": 10,
                "opacity": 1,
                "padding": 20,
                "position": "relative",
                "width": 100,
              }
            }
          >
            <View
              style={
                [
                  {
                    "borderRadius": 22.5,
                    "bottom": 5,
                    "height": 45,
                    "right": 5,
                    "width": 45,
                  },
                  {
                    "alignContent": "center",
                    "alignItems": "center",
                    "backgroundColor": "#999",
                    "display": "flex",
                    "justifyContent": "center",
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            >
              <Text>
                +
                2
              </Text>
            </View>
            <Image
              source={
                {
                  "uri": "https://external-preview.redd.it/Af65FrXSGlwOobPUnC3MNOI1D3xCCERFWoVFfhzpgio.png?format=pjpg&auto=webp&s=ed7b1feb0f8356ba51ea70f5b26b81c1868bc005",
                }
              }
              style={
                [
                  {
                    "borderRadius": 27.5,
                    "height": 55,
                    "right": 5,
                    "top": 5,
                    "width": 55,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
            <Image
              source={
                {
                  "uri": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQS7_zsiM5wfE2uac1B7Xk-jdVGpCKzrQqEwOaK9ak60O1iI4Don3BcWR_sOsumMQetlSw&usqp=CAU",
                }
              }
              style={
                [
                  {
                    "borderRadius": 30,
                    "height": 65,
                    "top": 10,
                    "width": 65,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
          </View>
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "width": 100,
              }
            }
          >
            SSDrink&Drive
          </Text>
        </View>
        <View
          style={
            [
              {
                "justifyContent": "center",
              },
              {
                "marginLeft": 20,
              },
            ]
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#ddd",
                "borderRadius": 50,
                "height": 100,
                "marginBottom": 10,
                "opacity": 1,
                "padding": 20,
                "position": "relative",
                "width": 100,
              }
            }
          >
            <Image
              source={
                {
                  "uri": "https://static.wikia.nocookie.net/breakingbad/images/4/40/Season_2_-_Skyler.jpg/revision/latest?cb=20090617160516",
                }
              }
              style={
                [
                  {
                    "borderRadius": 27.5,
                    "height": 55,
                    "right": 5,
                    "top": 5,
                    "width": 55,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
            <Image
              source={
                {
                  "uri": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQS7_zsiM5wfE2uac1B7Xk-jdVGpCKzrQqEwOaK9ak60O1iI4Don3BcWR_sOsumMQetlSw&usqp=CAU",
                }
              }
              style={
                [
                  {
                    "borderRadius": 30,
                    "height": 65,
                    "top": 10,
                    "width": 65,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
          </View>
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "width": 100,
              }
            }
          >
            My family
          </Text>
        </View>
        <View
          style={
            [
              {
                "justifyContent": "center",
              },
              {
                "marginLeft": 20,
              },
            ]
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#ddd",
                "borderRadius": 50,
                "height": 100,
                "marginBottom": 10,
                "opacity": 1,
                "padding": 20,
                "position": "relative",
                "width": 100,
              }
            }
          >
            <Image
              source={
                {
                  "uri": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQS7_zsiM5wfE2uac1B7Xk-jdVGpCKzrQqEwOaK9ak60O1iI4Don3BcWR_sOsumMQetlSw&usqp=CAU",
                }
              }
              style={
                [
                  {
                    "borderRadius": 27.5,
                    "height": 55,
                    "right": 5,
                    "top": 5,
                    "width": 55,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
            <Image
              source={
                {
                  "uri": "https://external-preview.redd.it/Af65FrXSGlwOobPUnC3MNOI1D3xCCERFWoVFfhzpgio.png?format=pjpg&auto=webp&s=ed7b1feb0f8356ba51ea70f5b26b81c1868bc005",
                }
              }
              style={
                [
                  {
                    "borderRadius": 30,
                    "height": 65,
                    "top": 10,
                    "width": 65,
                  },
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 3,
                    "position": "absolute",
                  },
                ]
              }
            />
          </View>
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "width": 100,
              }
            }
          >
            Whiskey Wednesday Gang
          </Text>
        </View>
      </View>
    </RCTScrollView>
    <View
      style={
        {
          "display": "flex",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "marginTop": 20,
          "overflowX": "scroll",
        }
      }
    >
      <Text
        style={
          {
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 12,
            "marginRight": "auto",
            "marginVertical": 8,
            "paddingHorizontal": 20,
          }
        }
      >
        Saved restaurants
      </Text>
      <View
        accessibilityRole="button"
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            [
              {},
            ]
          }
        >
          <Text
            style={
              [
                {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "margin": 8,
                  "textAlign": "center",
                },
                {
                  "color": "#ff6e6e",
                },
              ]
            }
          >
            See all
          </Text>
        </View>
      </View>
    </View>
    <RCTScrollView
      horizontal={true}
      style={
        {
          "display": "flex",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "marginTop": 8,
          "overflowX": "scroll",
        }
      }
    >
      <View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "marginLeft": 20,
              "opacity": 1,
            }
          }
        >
          <Image
            source={
              {
                "uri": "https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/05/01/15/lospolloshermanos.jpg?width=1200",
              }
            }
            style={
              {
                "borderRadius": 8,
                "height": 100,
                "width": 100,
              }
            }
          />
          <Text
            style={
              {
                "marginLeft": "auto",
                "marginRight": "auto",
                "marginTop": 8,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            Los Pollos Hermanos
          </Text>
        </View>
      </View>
    </RCTScrollView>
    <View
      style={
        {
          "height": 20,
        }
      }
    />
  </View>
</RCTScrollView>
`;
